lib/components/Table.js

```diff
@@ -144,7 +144,7 @@ Table.propTypes = {
    * Options to be passed to table pagination.
    */
   paginationProps: PropTypes.object,
-  rowData: PropTypes.array,
+  rowData: PropTypes.array.isRequired,
```

Comment:
columnData prop is also required, right?

Reply:
> columnData prop is also required, right?

It will work without errors even if we don’t  provide columnData.